ASSIGMENT/src/pages/aadhaar.astro

```diff
@@ -0,0 +1,64 @@
+---
+import MainLayout from '../layouts/MainLayout.astro';
+import Nav from '../components/Nav.astro';
+---
+
+<head>
+    <link rel="stylesheet" type="text/css" href="/assets/styles/aadhaar.css">
```

Comment:
this is not correct way. import css in front matter( `---` ) section

Reply:
corrected